src/components/molecules/widgets/WidgetWrapper.tsx

```diff
@@ -39,10 +39,10 @@ import { WidgetName } from '../../../models/WidgetName';
 interface IProps {
   widget: IWidgetBase;
   segmentText: string;
-  options?: any;
+  sizeOptions?: any;
```

Comment:
why `any`?

Reply:
fixed- changed to number and add property sizeOptions to WidgetTemplate

Reply:
Still `any`

Reply:
because of : (cardEditorDialog)
 const widgetComponent = !widget ? null : (
    <WidgetWrapper widget={widget} segmentText={store.newsFlashWidgetsMetaString} sizeOptions={sizeOptions.size} />
  );
when changing to 'number' it cannot assign undefined to type number.

Reply:
change `CardSizeOptions` and make size mandatory.
From
```
export interface CardSizeOptions {
  size?: number;
}
```

To
```
export interface CardSizeOptions {
  size: number;
}
```